### DISCUSSION TITLE
Advise on what API report to be used to import daily orders

### DISCUSSION THREAD

**Discussion by Gopikrish25** [amzn/selling-partner-api-models#4152] [Q&A] [answered]
I am currently assisting a client in integrating Amazon Seller Central with their new ERP system. In their legacy system, they had previously integrated using a custom-made application. This application could process reports and return files in a format readable by the legacy ERP.

The issue I've noticed is that they are using reports generated from the payments tab daily and importing them as sales orders. I believe this may not be a recommended practice.

Can someone explain if the above process is acceptable? In my experience with other implementations, orders are usually imported daily without charges.

Thanks in advance!!

**Comment by jawheral** [comment, marked as answer]
Hi @Gopikrish25,

There are multiple order reports that the Selling Partner API offers which you can check [here](https://developer-docs.amazon.com/sp-api/docs/report-type-values-order).

Depending on the business needs and use case, some reports could be more suitable than others. For example, if the goal is Invoicing then Invoicing Order reports would be more suitable (e.g [Requested or Scheduled Flat File Order Report (Invoicing)](https://developer-docs.amazon.com/sp-api/docs/report-type-values-order#requested-or-scheduled-flat-file-order-report-invoicing)) 

Without further information on the use case, we can't point out exactly which report you require. Though, it sounds that what you need might be Order Tracking so this [report](https://developer-docs.amazon.com/sp-api/docs/report-type-values-order#flat-file-orders-by-order-date-report) can be useful. 

I hope this helps!

Best,
Jawher
Selling Partner API Solutions Architect

**Reply by Gopikrish25** [reply]
Hi @jawheral  , Thanks for your reply our client sells products in the NA region, and the report [https://developer-docs.amazon.com/sp-api/docs/report-type-values-order#requested-or-scheduled-flat-file-order-report-invoicing] you mentioned is not available for the NA region 

Their legacy system is not an API-based integration, and in the new ERP system, they would like to implement an API-based integration It looks like the report that is being used currently [legacy] cannot be exported by API [please correct me if I'm wrong].

Just so you know, the reports that they are currently exporting manually and feeding into their legacy ERP system are attached. 

<img width="948" alt="image" src="https://github.com/user-attachments/assets/c85c3f1f-fd02-49e9-96f0-9094e56be65f">

Our client's main goal is to maintain all the daily order-related data and include charges such as FBA Fees, Shipping, and Commission rates and invoice them once the settlement has arrived [weekly] so they can track the finances, inventory, and sales trends accordingly.

I guess it would be a great start for me to export all the possible types of reports that can be exported from SP-API and figure out which suits best in this case.

I appreciate your response thanks again!
